### ISSUE TITLE
possible typo of epithet

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#992]
[Originally posted on GoogleCode (id 652) on 2011-08-03]

<b>(This is the template to report a data issue. If you want to report</b>
<b>another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/9284

<b>What data are incorrect or missing?</b>
?Saxifraga cespitosa subsp. exaratioides 

<b>What data are you expecting instead?</b>
The subsp. is spelled as exaratoides and exaratioides in both Tropicos and IPNI under all combinations of this epithet, so I don't know which spelling is correct.

<b>If applicable, please provide an authoritative source.</b>

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2011-08-04 15:44Z]

IPNI is very consistent and uses exaratioides in all combinations, while TROPICOS uses mainly exaratoides but occasionally exaratioides

I will ask but my feeling is that the proper spelling is exaratioides

Luc

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2011-08-04 17:36Z]

Here is the answer from K. Gandhi:

The original epithet spelling is: exaratoides (similar to S. exarata).

IPNI should be corrected.

Why some authors added an “I” to precede the suffix “-oides”?

They assumed that the stem is “exarat-” and that a connecting vowel is added to precede the suffix “-oides”.

But from Stearn’s Botanical Latin, it is cleare that the suffix “-oides” has 2 parts: “o” + “ides”.  And “o” belongs to the stem.

In other words, “exarat + o + ides = exaratoides

Kanchi
